egs/swbd/s5c/local/ctc/run_lstm.sh

```diff
@@ -22,7 +22,7 @@ norm_based_clipping=true
 # natural gradient options
 ng_per_element_scale_options=
 ng_affine_options=
-num_epochs=4
+num_epochs=2
```

Comment:
2 epochs is being used in TDNN+Cross entropy. So I am running
LSTM+cross-entropy with 4 epoch and LSTM+CTC with 2 epochs to account for
the frame-shifts.

Vijay

On Wed, Oct 7, 2015 at 10:50 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/ctc/run_lstm.sh
> <https://github.com/kaldi-asr/kaldi/pull/217#discussion_r41471484>:
>
> > @@ -22,7 +22,7 @@ norm_based_clipping=true
> >  # natural gradient options
> >  ng_per_element_scale_options=
> >  ng_affine_options=
> > -num_epochs=4
> > +num_epochs=2
>
> 2 epochs is good for tuning, but we may need to increase it later on for
> best results.. no problem to merge though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/217/files#r41471484>.
>